ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1093 @@
+oc create -f ejb-txn-remote-call/server/server-cr.yaml
+----
+
+If these commands are successful, the `oc get pod` command shows
+all the pods required for the quickstart, namely the quickstart client and two
+server pods, the {productName} Operator pod, and the PostgreSQL database pod
+that the application connects to.
+
+[source,sh,options="nowrap"]
+----
+NAME                                READY   STATUS      RESTARTS   AGE
+client-0                            1/1     Running     0          29m
+postgresql-f9f475f87-l944r          1/1     Running     1          22h
+server-0                            1/1     Running     0          11m
+server-1                            1/1     Running     0          11m
+wildfly-operator-5d4b7cc868-zfxcv   1/1     Running     1          22h
+----
+
+[#_running_on_openshift_verify_the_quickstarts]
+==== Running on OpenShift: Verify the Quickstarts
+
+The {productName} Operator creates routes that makes the applications accessible from 
+outoutside of the OpenShift environment. Run the `oc get route` command to find the location of the REST endpoint.
+An example of the output is:
+
+[source,sh,options="nowrap"]
+----
+NAME           HOST/PORT                                      PATH   SERVICES              PORT
+client-route   client-route-host-namespace.apps-crc.testing          client-loadbalancer   http
+server-route   server-route-host-namespace.apps-crc.testing          server-loadbalancer   http
+----
+
+[source,sh,options="nowrap"]
+----
+curl -s client-route-host-namespace.apps-crc.testing/remote-outbound-stateless | jq .
+curl -s client-route-host-namespace.apps-crc.testing/remote-outbound-notx-stateless | jq .
+curl -s client-route-host-namespace.apps-crc.testing/direct-stateless | jq .
+curl -s client-route-host-namespace.apps-crc.testing/remote-outbound-notx-stateful | jq .
+
+# To check failure resolution
+# verify the number of commits that come from the first and second node of the `server` deployments.
+# Two calls are needed, as each reports the commit count of different node.
+curl -s server-route-host-namespace.apps-crc.testing/commits
+curl -s server-route-host-namespace.apps-crc.testing/commits
+# Run the remote call that causes the JVM of the server to crash.
+curl -s client-route-host-namespace.apps-crc.testing/remote-outbound-fail-stateless
+# The system pauses to wait for recovery and the number of commits increases
+while true; do
+  curl -s server-route-host-namespace.apps-crc.testing/commits
+  curl -s server-route-host-namespace.apps-crc.testing/commits
+  I=$((I+1))
+  echo " <<< Round: $I >>>"
+  sleep 2
+done
+----
+
+//*************************************************
+// CD Release content only
+//*************************************************
+
+ifdef::EAPCDRelease[]
+
+// Basic management actions for the OpenShift cluster
+include::../shared-doc/cd-post-deployment-tasks.adoc
+
+endif::[]
```

Comment:
You should add other shared docs sections, such as the CodeReady one, otherwise you will get in trouble with QE. Please have a look at the README.adoc of the recent microprofile-config quickstart, which is the current and simple XP Quickstart to look at first, as a model of what you need here too.